### PR TITLE
[NETBEANS-54] Module Review

### LINKED ISSUE #67: Cleanup NetBeans license header for module html.lexer
Review and cleanup the license header in html.lexer

Oracle donated NetBeans to the Apache Software Foundation.

Your task now is to *replace* the old license header that has Oracle (and/or Sun Microsystems
and/or NetBeans) in it with the standard Apache Software Foundation license header.

For an empty file that supports headers you must *add* the Apache license header.

Note that you have to use the proper license header for Java, .properties or HTML files.
Look at the nearby files for templates and copy-paste the license header.

When done commit the changes with a commit message that starts with first line
"[NETBEANS-54] Module Review <module>" and a message body describing your review.

See a longer explanation about what we are doing here: https://cwiki.apache.org/confluence/display/NETBEANS/List+of+Modules+to+Review

Please announce in a comment when you have started work.

Files that need to be looked at in html.lexer:
```
html.lexer/src/org/netbeans/lib/html/lexer/HtmlElements.java
html.lexer/test/unit/data/testfiles/testInput.html.txt
html.lexer/test/unit/data/testfiles/testInput.html.txt.tokens.txt
html.lexer/test/unit/data/testfiles/testInput.rb.txt
html.lexer/test/unit/data/testfiles/testInput.rb.txt.tokens.txt

```

### PR DESCRIPTION
**Pull request by bondolo**
Switches HtmlElements.java to use ASF license.

Fixes #67

### CODE REVIEW

File: html.lexer/src/org/netbeans/lib/html/lexer/HtmlElements.java [hunk 1]
```diff
@@ -1,46 +1,22 @@
...
- * Version 2 license, then the option applies only if the new code is
- * made subject to such option by the copyright holder.
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
  */
+
```
Comment (emilianbold) [inline html.lexer/src/org/netbeans/lib/html/lexer/HtmlElements.java]: This extra new line is not necessary.

**Review by emilianbold — CHANGES_REQUESTED** [review]
You must also exclude in nbbuild/build.xml ("rat" task) the test data from html.lexer/test/unit/data/testfiles/**

**Comment by bondolo** [comment]
Should automatically resolve when you pull from apache repo

**Review by emilianbold — CHANGES_REQUESTED** [review]
I cannot review this since you did a bad rebase or something and I see 655 files changed.

A lot of commits appear as 'committed with Mike Duigou' which is a sign of rebase to me.

Please do a proper merge / rebase and push the new changes.

**Comment by emilianbold** [comment]
Closing this since you opened https://github.com/emilianbold/incubator-netbeans/pull/327

**Pull request closed without merging**